opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/VisitVaccineUtilTest.java

```diff
@@ -93,7 +121,8 @@ public void testGetInMemoryAlertsGetsOfflineAlerts(){
         DateTime dob = DateTime.now();
         List<Vaccine> issuedVaccines = new ArrayList<>();
 
-        VisitVaccineUtil.getInMemoryAlerts(vaccineSchedules, baseEntityId, dob, vaccineCategory, issuedVaccines);
+        List<Alert> inMemoryAlerts = VisitVaccineUtil.getInMemoryAlerts(vaccineSchedules, baseEntityId, dob, vaccineCategory, issuedVaccines);
         verify(vaccineSchedule, atLeastOnce()).getOfflineAlert(eq(baseEntityId), eq(dob.toDate()), eq(issuedVaccines));
+        Assert.assertTrue(inMemoryAlerts.contains(offlineAlert));
```

Comment:
Also check the size of list returned so that it's definite. If someone adds more offline alerts for the test. This test will still pass. If the method logic internally generates extra alerts, this test will still pass.